src/python/grpcio/grpc/_cython/_cygrpc/channel.pyx.pxi

```diff
@@ -382,7 +382,7 @@ cdef object _watch_connectivity_state(
           state.c_connectivity_completion_queue, <cpython.PyObject *>tag)
       state.connectivity_due.add(tag)
     else:
-      raise ValueError('Cannot invoke RPC: %s' % state.closed_reason)
+      raise ValueError('Cannot monitor channel state: %s' % state.closed_reason)
```

Comment:
Should we update the type of the exception to be `RuntimeError`?

Reply:
While I would agree if this were new code, someone could be depending on the particular type of this exception, so this could be considered a behavior change. Leaving this as it is for now.